actors/builtin/miner/miner_actor.go

```diff
@@ -1753,8 +1784,10 @@ func handleProvingDeadline(rt Runtime) {
 			// Release pledge requirements for the sectors expiring on-time.
 			// Pledge for the sectors expiring early is retained to support the termination fee that will be assessed
 			// when the early termination is processed.
-			pledgeDelta = big.Sub(pledgeDelta, expired.OnTimePledge)
-			st.AddInitialPledgeRequirement(pledgeDelta)
+			pledgeDelta = expired.OnTimePledge
+			st.AddInitialPledgeRequirement(pledgeDelta.Neg())
+			// Only record the amount we can actually unlock
+			pledgeDelta = st.UnlockInitialPledgeDeposits(pledgeDelta)
```

Comment:
Eeek this is getting hard to understand, I think due to the possible confusion between pledge requirements and coverage.